gfw_pixetl/decorators.py

```diff
@@ -79,9 +79,7 @@ def wrapper(*args, **kwargs):
             raise SubprocessKilledError("Process was killed")
         elif error:
             ex_type, ex_value, tb_str = error
```

Comment:
When unpacking unused data is a `_` a good idea? Just wondering why you kept the unused variables.
```suggestion
            _, ex_value, _ = error
```

Reply:
Good point, thanks!